python/mxnet/gluon/nn/conv_layers.py

```diff
@@ -673,7 +673,7 @@ def __init__(self, channels, kernel_size, strides=(1, 1, 1), padding=(0, 0, 0),
 class _Pooling(HybridBlock):
     """Abstract class for different pooling layers."""
     def __init__(self, pool_size, strides, padding, ceil_mode, global_pool,
-                 pool_type, count_include_pad=None, **kwargs):
+                 pool_type, layout, count_include_pad=None, **kwargs):
```

Comment:
Do we need to expose this option to user? Is there any use case?

Reply:
I'm not sure I understand - that option was already exposed to users, e.g. see here: https://mxnet.incubator.apache.org/api/python/gluon/nn.html#mxnet.gluon.nn.MaxPool1D
It just was not passed up to base class and to C++ because pooling supported only 1 layout.